examples/exec_graphql.py

```diff
@@ -43,10 +43,18 @@ def main():
 
     check_client(client)
 
-    response = client.execute_graphql(query='{ tdh_allIssues { id reference } }')
-    print("All TDH Issues:")
-    for issue in response.tdh_allIssues:
-        print(f"{issue.id} {issue.reference}")
+    try:
+        response = client.execute_graphql(query='{ tdh_allIssues { id reference } }')
+    except PortalAPIError as exc:
+        print_error(str(exc))
+        sys.exit(1)
+
+    if len(response.tdh_allIssues) > 0:
+        print("All TDH Issues:")
```

Comment:
At some point we'd like to maybe change print statements to `logging` calls?

Reply:
These are example scripts, and the output is for humans, definitely not for anything production or automated systems.